autoinstall/Pardus/preseed.cfg

```diff
@@ -0,0 +1,250 @@
+# To create a normal user account.
+d-i passwd/user-fullname string {{ new_user }}
+d-i passwd/username string {{ new_user }}
+# Normal user's password encrypted using a crypt(3) hash.
+d-i passwd/user-password-crypted password {{ vm_password_hash }}
+
+# The user account will be added to some standard initial groups. To
+# override that, use this.
+d-i passwd/user-default-groups string root wheel
+{% endif %}
+
+### Partitioning
+## Partitioning example
+# Specify a disk to partition.
+# For example, to use the first SCSI/SATA hard disk:
+d-i partman-auto/disk string /dev/{{ boot_disk_name }}
+# In addition, you'll need to specify the method to use.
+# The presently available methods are:
+# - regular: use the usual partition types for your architecture
+# - lvm:     use LVM to partition the disk
+# - crypto:  use LVM within an encrypted partition
+d-i partman-auto/method string lvm
+
+# You can define the amount of space that will be used for the LVM volume
+# group. It can either be a size with its unit (eg. 20 GB), a percentage of
+# free space or the 'max' keyword.
+d-i partman-auto-lvm/guided_size string max
+
+# If one of the disks that are going to be automatically partitioned
+# contains an old LVM configuration, the user will normally receive a
+# warning. This can be preseeded away...
+d-i partman-lvm/device_remove_lvm boolean true
+# The same applies to pre-existing software RAID array:
+d-i partman-md/device_remove_md boolean true
+# And the same goes for the confirmation to write the lvm partitions.
+d-i partman-lvm/confirm boolean true
+d-i partman-lvm/confirm_nooverwrite boolean true
+
+# You can choose one of the three predefined partitioning recipes:
+# - atomic: all files in one partition
+# - home:   separate /home partition
+# - multi:  separate /home, /var, and /tmp partitions
+d-i partman-auto/choose_recipe select atomic
+
+# This makes partman automatically partition without confirmation, provided
+# that you told it what to do using one of the methods above.
+d-i partman-partitioning/confirm_write_new_label boolean true
+d-i partman/choose_partition select finish
+d-i partman/confirm boolean true
+d-i partman/confirm_nooverwrite boolean true
+
+# Force UEFI booting ('BIOS compatibility' will be lost). Default: false.
+{% if firmware is defined and firmware == 'efi' %}
+d-i partman-efi/non_efi_system boolean true
+
+# Ensure the partition table is GPT - this is required for EFI
+d-i partman-partitioning/choose_label string gpt
+d-i partman-partitioning/default_label string gpt
+{% endif %}
+
+# This makes partman automatically partition without confirmation.
+d-i partman-md/confirm boolean true
+d-i partman-partitioning/confirm_write_new_label boolean true
+d-i partman/choose_partition select finish
+d-i partman/confirm boolean true
+d-i partman/confirm_nooverwrite boolean true
+
+# Add offical repo
+d-i apt-setup/local0/repository string deb http://depo.pardus.org.tr/pardus yirmibir main contrib non-free
```

Comment:
Please don't hard code Pardus codename here. If this online repo can't be used for `pkgsel/include`, I think you can use command `apt-add-repository` to add online repo in `preseed/late_command` where you can get the OS codename from /etc/os-release.

Reply:
Change it. Thanks